t/Util.pm

```diff
@@ -820,4 +824,65 @@ sub spawn_dns_server {
     return $server;
 }
 
+sub run_openssl_client {
+    my($opts) = @_;
+    my $port = $opts->{port} or croak("`port` is required!");
+    my $san = $opts->{san};
+    my $host = $opts->{host} // '127.0.0.1';
+    my $path = $opts->{path} // '/';
+    my $timeout = $opts->{timeout} // 2.0;
+    my $request = $opts->{request};
+    my $request_default = $opts->{request_default} // 0;
```

Comment:
```suggestion
    my $request_default = $opts->{request_default} // undef;
```
Assuming that this is a boolean, the custom is to represent false using an undef.